File: Modul-6/Py/PRAK605-2310817110006-Adrian_Bintang_Saputera.py
n = int(input("Input\n"))

print("Matriks A")
A = [list(map(int, input().split())) for _ in range(n)]

print("Matriks B")
B = [list(map(int, input().split())) for _ in range(n)]

hasil = [[0] * n for _ in range(n)]
for i in range(n):
    for j in range(n):
        for k in range(n):
            hasil[i][j] += A[i][k] * B[k][j]

print("\nOutput")
print("Matriks AXB")
for i in range(n):
    print(*hasil[i])

File: Modul-6/C/PRAK605-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int n;

  printf("Input\n");
  scanf("%d", &n);

  printf("Matriks A\n");
  int A[n][n];
  for (int i = 0; i < n; i++)
  {
    for (int j = 0; j < n; j++)
    {
      scanf("%d", &A[i][j]);
    }
  }

  printf("Matriks B\n");
  int B[n][n];
  for (int i = 0; i < n; i++)
  {
    for (int j = 0; j < n; j++)
    {
      scanf("%d", &B[i][j]);
    }
  }

  int hasil[n][n];
  for (int i = 0; i < n; i++)
  {
    for (int j = 0; j < n; j++)
    {
      hasil[i][j] = 0;
      for (int k = 0; k < n; k++)
      {
        hasil[i][j] += A[i][k] * B[k][j];
      }
    }
  }

  printf("\nOutput\n");
  printf("Matriks AXB\n");
  for (int i = 0; i < n; i++)
  {
    for (int j = 0; j < n; j++)
    {
      printf("%d ", hasil[i][j]);
    }
    printf("\n");
  }

  return 0;
}